htdocs/bookcal/availabilities_card.php

```diff
@@ -132,8 +136,29 @@
 
 	$triggermodname = 'BOOKCAL_AVAILABILITIES_MODIFY'; // Name of trigger action code to execute when we modify record
 
-	// Actions cancel, add, update, update_extras, confirm_validate, confirm_delete, confirm_deleteline, confirm_clone, confirm_close, confirm_setdraft, confirm_reopen
-	include DOL_DOCUMENT_ROOT.'/core/actions_addupdatedelete.inc.php';
+	$dateStartTimestamp = dol_stringtotime(GETPOST('start'));
```

Comment:
In dolibarr dates are not passed as a parameter into only one field. There are propaged into 3 or 6 fields (one field for each component). and then converted into a date using the 
dol_mktime(...the 6 components...). Date into the string has a format that depends on user language and is not predicable on some languages

You should ignore GETPOST('start') and use GETPSOT('startday'),  GETPSOT("sartmonth')...